Emby.Server.Implementations/IO/LibraryMonitor.cs

```diff
@@ -82,9 +82,7 @@ public void ReportFileSystemChangeBeginning(string path)
         public bool IsPathLocked(string path)
         {
             // This method is not used by the core but it used by auto-organize
-
-            var lockedPaths = _tempIgnoredPaths.Keys.ToList();
-            return lockedPaths.Any(i => _fileSystem.AreEqual(i, path) || _fileSystem.ContainsSubPath(i, path));
+            return _tempIgnoredPaths.Keys.Any(i => _fileSystem.AreEqual(i, path) || _fileSystem.ContainsSubPath(i, path));
```

Comment:
I'm not sure this is thread safe. I believe the point of the `ToList` call is to grab a "snapshot" of the data.